styles/index.css

```diff
@@ -2,6 +2,7 @@
 @import "./global/chip.css";
 @import "./global/reset.css";
 @import "./global/variables.css";
+@import ".sections/header.css";
```

Comment:
- [ ] Here, use `@import "./sections/header.css"` instead of  `@import ".sections/header.css"`. Using the `/` is important as if you intended to use a relative path, you should use `./` instead of `.`